client/resource_group/controller/controller.go

```diff
@@ -112,14 +117,17 @@ func NewResourceGroupController(
 	requestUnitConfig *RequestUnitConfig,
```

Comment:
Do we still need this parameter?

Reply:
I can't decide if we need to use the specified config on the client side, so I keep it.